.github/workflows/coverage.yml

```diff
@@ -0,0 +1,41 @@
+name: Test Coverage
+
+on:
+  workflow_dispatch:
```

Comment:
`workflow_dispatch` is intended to enable manual running of action. This does not seem to be working at the moment.

Reply:
I'd like to see if it would activate when merged to master. It's definitely something to look into. At the moment we can only use the "Re-run all jobs" option and sure that works. But it feels inefficient.

Reply:
Yea, so with the files split we'll be fine with "Re-run all jobs" since it will only run all jobs within this file :+1: 

Removing `workflow_dispatch`  :+1: